api/sentry/api_sentry.py

```diff
@@ -190,54 +237,32 @@ def sentry_rates(self, releases=[]):
 
 class DatabaseSentry:
 
-    def __init__(self):
+    def __init__(self, project_id='', platform='android'):
         print("DatabaseSentry.__init__()")
         super().__init__()
         self.db = Database()
-
-    # Filter out the non-production versions such as 9000
-    def _production_versions(self, version):
-        version = version.strip()
-        if version is None or version == '' or version == '9000':
-            return False
-        if "(" in version or ")" in version:
-            return False
-        if "org.mozilla.ios.Firefox" in version:
-            return False
-        parts = version.split('.')
-        return all(p.isdigit() for p in parts) and len(parts) > 0
-
-    # Get the beta and the release versions and all their
-    # dot releases.
-    def _all_new_production_dot_versions(self, versions):
-        major_versions = []
-        for version in versions:
-            parts = version.split('.')
-            major = parts[0]
-            major_versions.append(major)
-        major_versions = sorted(list(set(major_versions)), reverse=True)
-        major_versions = major_versions[:NUM_MAJOR_VERSIONS]
-        payload = []
-        for major_version in major_versions:
-            for version in versions:
-                if version.startswith(major_version+"."):
-                    payload.append(version)
-        payload = sorted(list(set(payload)), reverse=True)
-        print("Most recent {0} major versions:".format(NUM_MAJOR_VERSIONS))
-        print(payload)
-        return payload
+        self.project_id = project_id
+        self.platform = platform
 
     # Get the last two major versions
-    def report_version_strings(self, release_versions):
+    def report_version_strings(self, release_versions, latest_version):
```

Comment:
Wondering if we should this string manipulation work here or in utils better...

Reply:
I have moved `report_version_strings` as a utility of `SentryClient`.